### ISSUE TITLE
Get transactions from other accounts, not just a current account

### ISSUE THREAD

**Issue by flacle** [orson1282/arubabank_api#4]
Usually users have more than one account in their portfolio.

**Comment by orson1282** [comment]
Yeah so with the -b <account_number> arg it invokes the get_account_id function that searches the default portfolio for that account. So it can search for other accounts but only within the default portfolio.

A portfolio is defined as follows: Say you have an individual checking account with Aruba Bank and maybe a savings account. Those 2 accounts would be under one portfolio.

Eventually you get married, create a joint checking account and joint savings account, that's another portfolio, then after that you build a house, get a mortgage, that again goes in another portfolio. You get a kid, open a Cuenta ABC with them that goes into another portfolio and so on.

At the moment the api can only loop through accounts in the default portfolio, the default portfolio can be specified by the user when logged into their Aruba Bank Account.

There's a way to switch to another portfolio. Maybe we can get all portfolios and loop through them looking for the account specified with -b <account_number> then returning the transactions. Tonight I will look at which api call is used to change active portfolio.
